pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = [
+    "cmake>=3.19",
```

Comment:
Sounds good. This might be outdated. Be careful the link you send redirect to 0.9.5.dev8 while latest stable is 0.9.4. I would check https://scikit-build-core.readthedocs.io/en/stable/ instead of latest

Reply:
Thanks. Didn't realize this wasn't the stable docs.